install/playbooks/roles/dovecot/templates/apparmor.d/tunables/dovecot.cf

```diff
@@ -11,8 +11,7 @@
 
 # @{DOVECOT_MAILSTORE} is a space-separated list of all directories
 # where dovecot is allowed to store and read mails
-# 
+#
 
 # HomeBox specific
-@{DOVECOT_MAILSTORE}=@{HOME}/mails/ @{HOME}/mails/maildir
-
+@{DOVECOT_MAILSTORE}=@{HOME}/mails/ @{HOME}/mails/maildir
```

Comment:
Should the `@{HOMEDIRS}` variable be extended to `/home/users/` in `/etc/apparmor.d/tunables/home.d/` so that `@HOME` can take these into account? It doesn't seem to be the case right now.

Or if we want to really restrict and exclude the `/root` and `/home/*` home directories, maybe just not use `@{HOME}` here and set `/home/users/*/mails/ /home/users/*/mails/maildir/`? Are system users or root expected to have mail delivered by dovecot?

IMHO, there should be a newline at the end of the file : just to be safe whatever apparmor is doing with the includes.

Reply:
I will test this, it make sense.

Reply:
Finally, I think it is better to restrict it to /home/users/*, and exclude /home/*.
It can be useful to have more restrictive rules for accounts in /home/users than those in home/. We can change it later, as it may have effects on other AppArmor rules we will need to check.
Also, I want to merge this asap to come back on two branches.

Reply:
I agree. Let's restrict it in another commit.